scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -4124,6 +4124,14 @@ def run(self):
                 self.__report_container_metrics,
             )
         )
+
+        server_link = self._global_config.scalyr_server
+        if server_link == "https://agent.scalyr.com":
+            server_link = "https://app.scalyr.com"
+        elif server_link == "https://upload.eu.scalyr.com":
+            server_link = "https://app.eu.scalyr.com"
+        global_log.info("To see your logs go to %s" % server_link, force_stdout=True)
```

Comment:
I'd actually like to make this even more fancy... we need to create a link that actually shows the logs for this pod on the events page.  I would suggest we move this out of this PR and follow up in another.  I would suggest looking at creating a search that restricts to the log file equaling the agent.log and the server host for this agent.

Also, some folks out there have their `scalyr_server` set to `www.scalyr.com` or `api.scalyr.com`.  I think what we do here is first check if the server host contains `eu.scalyr.com`.  If so, emit the eu version.. otherwise the www.scalyr.com.   It would be good to handle the staging environments as well.

Take a look at the `status -v` output the agent produces.  We do generate a link there as well.

Reply:
The `status -v` output has exactly what you are asking for as far as I can tell, so just a quick copy over should do it. Its minor enough that another PR might be a bit much.